app/views/downloads/download_mac.html.erb

```diff
@@ -17,6 +17,10 @@
   <p>Install <a href="https://brew.sh/">homebrew</a> if you don't already have it, then:<br>
   <code>$ brew install git</code></p>
 
+  <h3>MacPorts</h3>
```

Comment:
tbf i'm not sure macports is "an important package manager for macOS", as homebrew seems to be the 1st choice for almost everyone (see google trend as well  https://trends.google.com/trends/explore/TIMESERIES/1643798400?hl=pt-PT&tz=0&cat=32&q=%2Fm%2F0fq175b,%2Fm%2F05_6wm&sni=3 )

but as long as the list doesn't get huge, I guess it's a valid option to download git as well